src/main/java/com/exadel/ipromise/entity/User.java

```diff
@@ -1,19 +1,29 @@
 package com.exadel.ipromise.entity;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class User {
 
+    @JsonProperty("user_id")
     private int userId;
-    private String nickname;
+
+    @JsonProperty("nick_name")
+    private String nickName;
+
+    @JsonProperty("email")
     private String email;
-    private int promiseId;
 
-    public User() {}
+    @JsonProperty("password")
```

Comment:
In case you have exactly the same name for Object field and the field name in DB - not need use @JsonProperty() - it's just for cases when names are different